### PR TITLE
Merge latest updates from Infrastructure.

### PR DESCRIPTION
**Pull request by stephenbelt**
This PR is to merge the latest updates from Infrastructure.  2 updates of note:

1.  Examples are added for all api routes.
2. Namespaces are noted to NOT allow leading and trailing white space.

### CODE REVIEW

**Review by jasnames — CHANGES_REQUESTED** [review]
There seems to be a mapping and duplication issue. This PR adds folders for content and system that duplicate the content in tenant. We only want the files in the tenant folder.

content-service-blog = tenant-service-blog
system-features = tenant-features
system-namespaces = tenant-namespaces
system-root-access-control = tenant-root-access-control
system-tenant = tenant-tenants

Other than that, just a few minor grammar updates and I need to know the ADH values for a few of these items for when I migrate the content over.

File: content/api-reference/tenant/tenant-service-blog.md [hunk 1]
```diff
@@ -30,19 +30,28 @@ GET /api/v1/ServiceBlog/Entries
...
-    "IsDeleted": true,
-    "Id": 0,
-    "TimePosted": "2019-08-24T14:15:22Z",
-    "Title": "string",
-    "Content": "string",
-    "Author": "string"
+    "Id": 2147483586,
+    "TimePosted": "2021-11-22T15:54:34.477Z",
+    "Title": "Blog Entry for Example 1",
+    "Content": "This entry is here for to show the example.",
```
[the remark this answers is not in the document]
Comment (stephenbelt) [reply content/api-reference/tenant/tenant-service-blog.md:46]: Update applied to Infrastructure source.

File: content/api-reference/tenant/tenant-namespaces.md [hunk 2]
```diff
@@ -21,49 +21,44 @@ GET /api/v1/Tenants/{tenantId}/Namespaces
...
-          "AccessType": 0,
-          "AccessRights": 0
-        }
-      ]
-    },
-    "RegionId": "string",
-    "InstanceId": "string"
+    "Id": "Namespace1",
+    "Region": "WestUS",
+    "Self": "https://dat-b.osipi.com/api/v1/tenants/00c97c8b-8f60-4f29-af71-3178c414e7a0/namespaces/Namespace1",
```
[the remark this answers is not in the document]
Comment (stephenbelt) [reply content/api-reference/tenant/tenant-namespaces.md:45]: https://euno.datahub.connect.aveva.com/ is the base address for Europe in ADH Prod.

**Comment by stephenbelt** [comment]
Due to the mapping issue identified by @jasnames, I am going to abandon this PR.  A new PR is open in DocuPipe repos to repair the mappings, and when that is complete, I will open a new PR here.

**Pull request closed without merging**